device-types/Intel/HNS2600TPR.yaml

```diff
@@ -0,0 +1,23 @@
+---
+manufacturer: Intel
+model: HNS2600TPR
+slug: intel-hns2600tpr
+u_height: 0.5
+comments: |
+  [HNS2600TPR Specifications on intel.com](https://ark.intel.com/content/www/us/en/ark/products/88297/intel-compute-module-hns2600tpr.html)
+subdevice_role: child
+inventory-items:
+  - name: S2600TPR
+    label: Intel Server Board S2600TPR
+    manufacturer: Intel
+interfaces:
+  - name: GigabitEthernet1
+    type: 1000base-t
+  - name: GigabitEthernet2
+    type: 1000base-t
+  - name: IPMI
+    type: 1000base-t
+    mgmt_only: true
+console-ports:
+  - name: Rear VGA
```

Comment:
This is not what this is intended for.. This is intended for console ports only. Please remove the VGA port